noxfile.py

```diff
@@ -177,6 +177,34 @@ def test_plugins(session, install_cmd):
             run_pytest(session)
 
 
+# run only joblib_launcher plugging
+@nox.session(python="3.6")
+def test_joblib(session):
+    install_cmd = ["pip", "install"]
+    session.install("--upgrade", "setuptools", "pip")
+    install_pytest(session)
+    install_hydra(session, install_cmd)
+    # Install all supported plugins in session
+    plugin = {
+        "name": "hydra_joblib_launcher",
+        "path": "hydra_joblib_launcher",
+        "module": "hydra_plugins.hydra_joblib_launcher",
+    }
+
+    cmd = list(install_cmd) + [os.path.join("plugins", plugin["path"])]
+    session.run(*cmd, silent=SILENT)
+
+    # Test that we can import Hydra
+    session.run("python", "-c", "from hydra import main", silent=SILENT)
+
+    # Test that we can import joblib plugin
+    session.run("python", "-c", "import {}".format(plugin["module"]))
+
+    # Run tests for joblib plugin
+    session.chdir(os.path.join(BASE, "plugins", plugin["path"]))
+    run_pytest(session)
```

Comment:
Rather than triggering the build many times, you could just a put this `run_pytest` call under a for loop, no?

Reply:
You can use:

```python
session.run("pytest", "--verbose")
```

to see the names of the tests.

Reply:
> 
> 
> Rather than triggering the build many times, you could just a put this `run_pytest` call under a for loop, no?

We tried it initially and it wouldn't repro. This suggests kind of interaction between different tests on the same host is causing it.

Reply:
I put a loop on `run_pytest` for the plugin and enabled verbose.

But I suspect putting verbose might make the issue disapears, let's see with this batch of CI run. For the loop thing, the interaction is still the same at least for the first run so it is worth trying.

Reply:
Ok I was able to reproduce with verbose here: https://app.circleci.com/jobs/github/facebookresearch/hydra/9226/parallel-runs/0/steps/0-105